lib/shared/types/src/types/config/models/project.ts

```diff
@@ -25,6 +25,7 @@ export class Project<IdType = string> {
                 primary: string
                 secondary: string
             }
+            poweredByPosition?: 'center' | 'left' | 'right'
```

Comment:
kind of a nitpick, but could we change this to `alignment`? "position" makes me think top/bottom/left/right, but this seems like it's actually controlling the alignment along the bottom?

Reply:
yup, will update